app/src/main/java/sopt/org/millie/presentation/search/SearchPostScreen.kt

```diff
@@ -0,0 +1,65 @@
+package sopt.org.millie.presentation.search
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import sopt.org.millie.core.designsystem.theme.MillieTheme
+
+@Composable
+fun PostItem(
+    postImageUrl: String,
+    title: String,
+    description: String,
+    writerUrl: String,
+    writerName: String,
+    writerDate: String,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .clip(RoundedCornerShape(8.dp))
+            .background(color = MillieTheme.colors.white),
+    ) {
+        MillieImage(
+            imageUrl = postImageUrl,
+            modifier = modifier
+                .align(Alignment.TopStart)
+                .size(91.dp, 132.dp),
+        )
+        Column(
+            modifier = Modifier
+                .padding(start = 107.dp, end = 14.dp, top = 16.dp, bottom = 15.dp),
+            verticalArrangement = Arrangement.spacedBy(10.dp),
+        ) {
+            PostBookInfo(title, description)
+            PostWriterInfo(writerUrl, writerName, writerDate)
```

Comment:
```suggestion
            PostBookInfo(title = title, description = description)
            PostWriterInfo(writerUrl = writerUrl, writerName = writerName, writerDate = writerDate)
```
파라미터 여러개일 때는 파라미터명 작성해줍시다 ㅎㅎ